RevenueCatUI/CustomerCenter/Views/NoSubscriptionsView.swift

```diff
@@ -47,34 +47,29 @@ struct NoSubscriptionsView: View {
     }
 
     var body: some View {
-        let background = Color.from(colorInformation: appearance.backgroundColor, for: colorScheme)
-        let textColor = Color.from(colorInformation: appearance.textColor, for: colorScheme)
-
         let fallbackDescription = "We can try checking your Apple account for any previous purchases"
 
-        ZStack {
-            if background != nil {
-                background.edgesIgnoringSafeArea(.all)
-            }
-            VStack {
+        List {
+            Section {
                 CompatibilityContentUnavailableView(
                     self.configuration.screens[.noActive]?.title ?? "No subscriptions found",
                     systemImage: "exclamationmark.triangle.fill",
                     description:
                         Text(self.configuration.screens[.noActive]?.subtitle ?? fallbackDescription)
-
                 )
+            }
 
-                Spacer()
-
+            Section {
```

Comment:
Not for this PR! But we still need to make the SDK respond to changes in the `noActive` screen paths right? I believe right now that's mostly ignored....

Separately, I'm also wondering if the `WrongPlatformView` should also be a separate "screen"... but we can work on that later if needed as well.

Reply:
oh right, forgot about that 😮‍💨